src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java

```diff
@@ -62,6 +62,7 @@ public enum DocumentTag {
     INTERNAL_APPELLANT_FTPA_DECIDED_LETTER("internalAppellantFtpaDecidedLetter", CaseType.ASYLUM),
     INTERNAL_HO_FTPA_DECIDED_LETTER("internalHoFtpaDecidedLetter", CaseType.ASYLUM),
     INTERNAL_HEARING_REQUIREMENTS_UPDATED_LETTER("internalHearingRequirementsUpdatedLetter", CaseType.ASYLUM),
+    INTERNAL_MAINTAIN_CASE_LINKS("internalMaintainCaseLinks", CaseType.ASYLUM),
```

Comment:
Can we have 'INTERNAL_MAINTAIN_CASE_LINKS_LETTER' just to stay consistent with other tags?